rs/rosetta-api/icp/tests/system_tests/common/utils.rs

```diff
@@ -205,3 +210,27 @@ pub async fn update_neuron(agent: &Agent, neuron: ic_nns_governance_api::pb::v1:
     .unwrap();
     assert!(result.is_none(), "Failed to update neuron: {:?}", result);
 }
+
+pub async fn get_pending_proposals(
```

Comment:
We should have this function in the rosetta client as well. There already exists a `call` functionality in the rosetta client. We should extend it by the option of getting pending proposals.

Reply:
Done.